lucid/optvis/render.py

```diff
@@ -41,7 +41,7 @@
 
 def render_vis(model, objective_f, param_f=None, optimizer=None,
                transforms=None, thresholds=(512,),
-               print_objectives=None, verbose=True,):
+               print_objectives=None, verbose=True, display_rate=10):
```

Comment:
That's an understandable wish for more/different ways to watch the optimization as it progresses. We are actively rethinking lucid's approach here in #17, which I'd invite you to comment on if you're interested!

Reply:
That being said, I'd prefer not to merge these specific API changes in `render.py` right now. To keep the record of your contribution intact I'd suggest undoing the changes in this file, so I can directly merge. (Rather than me cherry picking your first commit.)

Reply:
@ludwigschubert  Sorry, only the first commit was intended to be merged. Further changes I did for myself. I'll undo the changes now

Reply:
Updated according to the request.
Thanks for the amazing lib!